crates/uv-python/src/cpuinfo.rs

```diff
@@ -0,0 +1,24 @@
+//! Fetches CPU information.
+
+use anyhow::Error;
+#[cfg(target_os = "linux")]
+use procfs::{CpuInfo, Current};
+
+#[cfg(target_os = "linux")]
+pub(crate) fn detect_hardware_floating_point_support() -> Result<bool, Error> {
```

Comment:
The docstring should mention something about arm, afaik this instruction set does not exist on x86

Reply:
That's true. I will add the documentation.